root/static/scripts/edit/URLCleanup.js

```diff
@@ -1678,7 +1678,7 @@ const CLEANUPS = {
       }
       return false;
     },
-  },
+  },/* */
```

Comment:
Did you forget to write a comment, or is this something else?

Reply:
Oops, removed in https://github.com/metabrainz/musicbrainz-server/pull/1457/commits/e5213eab8acacb01a9a109682fda6f1570e70388